protomongo/protomongo.go

```diff
@@ -132,3 +135,15 @@ func oneofNames(pb descriptor.Message) map[string]bool {
 	}
 	return names
 }
+
+const (
+	tagPrefix = "PBTag_"
+)
+
+func TagToElementName(tag int) string {
```

Comment:
`tagToElementName`

Reply:
I actually need this exported - for https://github.com/dataform-co/dataform-co/pull/3622